### DISCUSSION TITLE
Install Guidance Needed

### DISCUSSION THREAD

**Discussion by cheme75** [koush/scrypted#238] [Q&A]
I have docker already running with wsl2 on an old i5 PC/win10 machine, I added the docker-wyze-bridge since I have hoobs on another old machine running lubuntu for a few wyze plugs and bulbs. The wyze bridge has been working well once I learned how to get the bridge config and ffmeg plugin setup correctly with a good deal of help from mrlt8 - who recently suggested I try scrypted. So here I am here and its not exactly clear what I need to do to get this setup on my docker PC - the wyze-bridge was simple to edit the docker-compose.yml per mrlt8 suggestions, and then copy and paste the docker-compose commands. I apologize if I missed the info for scrypted to be added to docker on a win machine, but it all looks like nas installs, so maybe someone can point me to the appropriate way to install this - I assume I need to pull an image somehow and then edit the yml file appropriately - I have done some seaching but nothing directly addresses running this and the wyze bridge together on docker - I'm just not clear on how to have multiple things setup and running in docker. Kind of disappointing to get no replies, making me think this is not well supported.

**Comment by ggravlingen** [comment]
The problem isn't this library but how WSL2 communicates through Docker. I gave it a go a while ago but gave up. Just found this, it might lead you in the right direction: https://github.com/microsoft/WSL/issues/4983#issuecomment-602487077

**Reply by cheme75** [reply]
Edit: seems you were just a few steps ahead of me. I got container setup and scrypted runs, but I can’t access the UI. Seems like it might be wsl2 related. Do will chk your link tomorrow. 

Not sure how that helps me figure out how to run this along with the wyze-bridge within docker which is running the wyze bridge just fine. What I need is some examples of how to setup the yml file to add scrypted so I can run both without screwing up what is working well now.

**Comment by cheme75** [comment]
@koush any help to get to the UI? I got it setup in a docker container on win10 - shows it s running. Allowed ports 10443 and 11080 in firewall, no luck browser fails to open page. Ran the PS script I saw in the install info, added 10443 and 11080 to the ports. changed ip to the static ip of the pc, seems to run script but window exits quickly. return to browser and still error. Edge says localhost refused to connect.

**Comment by cheme75** [comment]
Gave up on a win install - went with docker on old lubuntu netbook and I seem to be gaining some success. I got the doorbell cam streaming on homekit fairly easy but had some hassels with my 2 V2 cams - finally noticed transcoding was not selected for the DB cam so I removed it from both v2 and finally got those to stream now. lots more to tweak I guess - but at least I finally got some success!
